plot_multi_waveforms.py

```diff
@@ -0,0 +1,28 @@
+import make_waveforms
+import pylab as pl
+import numpy as np
+
+
+hp,hc= make_waveforms.make_tidal_waveform(mass1= 10, mass2 = 8, s1z=0, approx = "SpinTaylorT4")
```

Comment:
You are hardcoding the values of `hp` and `hc`. Instead, you should write a function that will take in the values of the set of parameters that are required to make the waveforms. that way every time you want to run the code with different parameters you will not have to change the code. Changing the code will mean you will have to again add, commit and push in git, again create pull request, and again this process.